### PR TITLE
Feature/track things more closely

### PR DESCRIPTION
**Pull request by hedvigoscar**

### CODE REVIEW

File: src/utils/tracking.ts [hunk 1]
```diff
@@ -1,10 +1,32 @@
+import { CookieStorage } from 'cookie-storage'
+import * as merge from 'deepmerge'
+
+const cookie = new CookieStorage()
```
Comment (palmfjord) [inline src/utils/tracking.ts]: this'll only work client-side though. Is there any time we may want tracking on the server or no?

Comment (hedvigoscar) [reply src/utils/tracking.ts]: I see your point but I'm not sure if we ever want to do 'this kind' of tracking from SSR. What do you think?

Comment (palmfjord) [reply src/utils/tracking.ts]: lets leave it for now 👍 obviously it wont work at all so i think its ok

**Comment by hedvigoscar** [comment]
@palmfjord I think this is ready for re-review 🙂

File: src/containers/SessionContainer.test.tsx [hunk 2]
```diff
@@ -23,6 +23,7 @@ jest.mock('../client/apolloClient', () => ({
     },
   },
 }))
+jest.mock('uuid/v4', () => () => '1')
```
Comment (palmfjord) [inline src/containers/SessionContainer.test.tsx]: is this needed now when we did `__mocks__.*`?

Comment (hedvigoscar) [reply src/containers/SessionContainer.test.tsx]: Doubt it! Lets check
```suggestion
```

**Pull request merged**